client/src/boot/cms/index.js

```diff
@@ -1,8 +1,28 @@
 /* global window */
 import registerComponents from './registerComponents';
 import registerReducers from './registerReducers';
+import { combineReducers, createStore } from 'redux';
+import Injector from 'lib/Injector'; // eslint-disable-line
 
 window.document.addEventListener('DOMContentLoaded', () => {
   registerComponents();
   registerReducers();
+
+  // Tell Injector to tag the dom when it's ready (see admins' `boot/index`)
+  Injector.ready(() => {
+    if (window.jQuery) {
```

Comment:
what about a `const { jQuery: $, ss } = window;` ?

Reply:
I just lifted this from `admin`. I considered it was probably fine to match.